src/Token.h

```diff
@@ -82,14 +82,19 @@ class Token {
     return *this;
   }
 
-  Token& flagNA(std::string NA) {
-    if ((size_t) (end_ - begin_) != NA.size())
-      return *this;
+  Token& flagNA(std::vector<std::string> NA) {
 
-    if (strncmp(begin_, &NA[0], NA.size()) != 0)
-      return *this;
+    std::vector<std::string>::iterator it;
+    for (it = NA.begin(); it != NA.end(); ++it) {
+      if ((size_t) (end_ - begin_) != (*it).size())
+        continue;
+
+      if (strncmp(begin_, &(*it)[0], (*it).size()) == 0) {
```

Comment:
I think `it->size()` would be a bit simpler, and I assume you can simplify the other expression similarly. (`it->data()` maybe?)